assets/chat/js/vote.js

```diff
@@ -93,6 +94,17 @@ class ChatVote {
     }
   }
 
+  canCastVote(timestamp) {
+    if (this.vote) {
+      return (
+        new Date(timestamp).getTime() -
+          (this.vote.start.getTime() + this.vote.time + VOTE_END_TIME) <
```

Comment:
```suggestion
          (this.vote.start.getTime() + this.vote.time) <
```

There's a bug where users can now cast votes during the cooldown. I think this change fixes it without breaking anything else, but I'd appreciate if you double-checked.